MekHQ/src/mekhq/campaign/mission/AtBScenario.java

```diff
@@ -2041,11 +2041,7 @@ public void removeBotForce(int i) {
     }
 
     public int getNumBots() {
-        if (isCurrent()) {
-            return botForces.size();
-        } else {
-            return botForceStubs.size();
-        }
+        return (getStatus().isCurrent() ? botForces : botForceStubs).size();
```

Comment:
Cute, but less readable.

Reply:
Forgot to revert that after testing for readability, reverted.